SimPEG/Maps.py

```diff
@@ -387,6 +387,300 @@ def nP(self):
         return self._nP
 
 
+class Tile(IdentityMap):
+    """
+        Mapping for tiled inversion
+    """
+
+    nCell = 27  # Number of neighbors to use in averaging
```

Comment:
just out of curiosity... where did this come from? (27 just seems oddly specific) :)

Reply:
Short answer : 3**3 = 27
It's a default to cover the case where a cell in the local problem is slightly offseted from the global mesh, so the maximum number of neighbors to worry about is a 3x3x3 block